doc/development_workflow.md

```diff
@@ -0,0 +1,60 @@
+# Development process in Status Open Bounty
+
+We have a continuously deployed version tracking the `develop` branch live at [test environment](https://openbounty.status.im:444). It uses the [Ropsten](https://ropsten.io/) Ethereum testnet. Any one is welcome to use it and contribute to Open Bounty.
+ 
+Any help is greatly appreciated!
+Currently we use two projects -  `Pipeline For Issues` and `Pipeline For Pull Requests`, issues and pull requests in our repository are passing through all or several stages described below.
+
+## Pipeline For Issues
+
+Issues are added to this project by product owner.
+
+#### To define
+This is backlog for all features/issues/enhancements which we want to include to development process. 
+
+All issues here should be marked with:
+* type - `bug`, `tech-debt`,` enhancement`, `proposal` labels
+* priority - `Prio: high`, `Prio: med`, `Prio: low` labels
+
+#### In Bounty
+The store with issues which are open bounties. When we put funds to issue and it shows up in [status open bounty](https://openbounty.status.im), we move issue here.
+
+#### Defining
+The column is intended for issues not completely clear or for features, that should be splitted to smaller issues in order to go ahead.
+After defining all issues that are intended to develop should have size label(`Size: S` - 2-3 hours, `Size: M` - 6-8 hours, `Size: L` - 8-24 hours)
+#### To design
+It uses for issues that are already defined and require designing process.
+#### Designing
+It shows up that issues are currently in designing process.
+#### To develop
+It stores issues which are ready for development.
+They are explained, clear, designed and **small enough to create one pull request per issue.**
+#### Developing
+This is for issues that are currently developing, so pull requests assosiated with issues have to be placed in `Pipeline for Pull Request` project. 
+#### Done
+It stores issues with merged to `master` pull requests.
+## Pipeline For Pull Requests
+
+#### Developing
+Contains all open pull requests (hereinafter PRs) that already assosiated with issues.
+#### Contributor
+It keeps all PRs from external contributors which should pass `Reviewing` stage.
```

Comment:
Not sure to get the point of this column. What is the difference with `Developing`?

I think that we should try to treat as much as possible internal and external work, unless we have a different specific workflow, but reviewing is something required for both internal and external pull requests.

Reply:
Okay, I get it now from something from below. How about calling it`Reviewing, waiting for contributor`? And we would also have `Reviewing`, the normal one.

Reply:
`Blocked` may also be a more generally usable label that we could add to PRs waiting for adjustments.

Reply:
@pablodip as you wish, let's call it `Reviewing, waiting for contributor`

@martinklepsch agree with `Blocked`, will add info

Reply:
Should we use a label or a column for `Blocked`? I like more a column so that we can visualise them better in the flow, and because when something is blocked it isn't being neither developed nor reviewed.

Reply:
Fine with either. :)

Reply:
As for me ,'Blocked' label is more convenient jut because it can be applied fir issues and for PRs. Two columns in 2 pipelines for me looks a bit redundant
@pablodip ?

Reply:
To me reusability shouldn't guide whether label or column, but actual stages in the flow of the pipeline. Anyway, let's go for label if you like it more and we can see how it works once we use it. :)